Test/dafny0/RuntimeTypeTests0.dfy.expect

```diff
@@ -2,12 +2,18 @@
 Dafny program verifier finished with 4 verified, 0 errors
```

Comment:
Where are we testing the new `NeedsCustomReceiver` logic?

Reply:
Added `comp/Variance.dfy` for that, but it only ensures that result works and not that `NeedsCustomReceiver` is minimal.